### PR TITLE
SCT-1147 Get sphinx autodoc working.

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,10 @@
+build-docs:
```
Comment (jayrbolton) [inline Makefile]: Maybe you are already aware, but you can automatically host docs under `/docs` with github pages by enabling it under the repo settings. There has to be a `docs/index.html` : https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch

File: docsource/index.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.. JGI/KBase ID Mapping Service documentation master file, created by
```
Comment (jayrbolton) [inline docsource/index.rst]: Here is a sphinx docs theme that I recommend: https://github.com/rtfd/sphinx_rtd_theme

**Comment by jayrbolton** [comment]
Just had a couple comments, no requests

File: Makefile (hunk 1 above)
Comment (MrCreosote) [reply Makefile]: I'm not aware, thanks. This is all internal API documentation so I'm not actually planning to host it anywhere as of yet.

File: docsource/index.rst (hunk 2 above)
Comment (MrCreosote) [reply docsource/index.rst]: Cool. If I ever put the docs up somewhere that looks like a good choice.

**Pull request merged**